packages/console/src/scss/_underscore.scss

```diff
@@ -38,6 +38,10 @@
   font: var(--font-subhead-cap-small);
 }
 
+@mixin image-block {
+  display: block;
+}
```

Comment:
display: block vs. @include _.image-block? feels like no need to define an extra mixin?

Reply:
Add `image-block` to clarify that this is a specific functionality.